packages/react-components/react-provider/src/components/FluentProvider/useFluentProvider.ts

```diff
@@ -3,6 +3,8 @@ import {
   ThemeContext_unstable as ThemeContext,
   useFluent_unstable as useFluent,
   useOverrides_unstable as useOverrides,
+  useCustomStyleHooks_unstable as useCustomStyleHooks,
+  CustomStyleHooksContextValue_unstable as CustomStyleHooksContextValue,
 } from '@fluentui/react-shared-contexts';
 import type { ThemeContextValue_unstable as ThemeContextValue } from '@fluentui/react-shared-contexts';
```

Comment:
You should `import type` `CustomStyleHooksContextValue_unstable`
```suggestion
  useCustomStyleHooks_unstable as useCustomStyleHooks
} from '@fluentui/react-shared-contexts';
import type {
  CustomStyleHooksContextValue_unstable as CustomStyleHooksContextValue,
  ThemeContextValue_unstable as ThemeContextValue
} from '@fluentui/react-shared-contexts';

```